framework/deproxy_client.py

```diff
@@ -613,7 +610,9 @@ def handle_read(self):
                 elif isinstance(event, TrailersReceived):
                     trailers = self.__headers_to_string(event.headers)
                     response = self.active_responses.get(event.stream_id)
-                    response.parse_text(str(response.headers) + "\r\n" + response.body + trailers)
+                    response.parse_text(
+                        str(response.headers) + "\r\n" + response.body + trailers + "\r\n"
+                    )
```

Comment:
I suggest reworking the logic here:
```python3
                elif isinstance(event, TrailersReceived):
                    response = self.active_responses.get(event.stream_id)
                    for trailer in event.headers:
                        response.trailer.add(trailer[0].decode(), trailer[1].decode())
```
I think that rebuilding the full message is not correct. We can just add trailer.

Reply:
okay.